Gems/ROS2/Code/Source/ROS2SystemComponent.cpp

```diff
@@ -148,6 +148,10 @@ namespace ROS2
         }
     }
 
+    float ROS2SystemComponent::GetExpectedSimulationLoopTime() const {
+        return  static_cast<float>(m_simulationClock.GetMedianExecutionTime()) / 1e6;
+    }
```

Comment:
standards: brace placement (as before).

It might also be worth switching to `aznumeric_cast` too